src/templates/layouts/storyListing/index.tsx

```diff
@@ -51,44 +51,46 @@ export function StoryListing({
     )
 
   return (
-    <div key={id} className="usa-grid usa-grid-full">
-      {/* Widget coming from vets-website */}
-      <nav
-        data-template="navigation/facility_sidebar_nav"
-        aria-label="secondary"
-        data-widget-type="side-nav"
-      ></nav>
+    <div id="content" className="interior">
```

Comment:
can we put this div inside of `[[...slug]]` so it only needs added in the one place? i do see it present on the newsStory layout already

Reply:
we may want to move `<main>` out of `wrapper.tsx` into `[[...slug]].tsx` also but that doesn't have to be done in this PR

Reply:
agreed, moved main and content divs to slug